deploy/examples/create-external-cluster-resources.py

```diff
@@ -319,7 +319,9 @@ def gen_arg_parser(cls, args_to_parse=None):
             "--cluster-name", default="", help="Ceph cluster name"
         )
         common_group.add_argument(
-            "--namespace", default="", help="Namespace where CephCluster is running"
```

Comment:
Renaming arguments is usually a breaking change. How will users of the existing `--namespace` flag be impacted? Maybe we don't need to rename it?

Reply:
I don't think so there would be any problem in replacing the flag name as in
For existing customers:
If they re-run the script for the upgrade they need to pass a new flag `--cluster-namesapce` and other than that import script will update it automatically,

If there is some automation used by the end user then only it might impact but there wont be any other impact I can look for

Reply:
It looks like there are two independent changes in this PR:
1. Rename the flag
2. Create the namespace

For the first item, could you provide more background for why the flag needs to be renamed? Like you said, it might impact user automation. It's probably a small impact, but we need to be clear and intentional about potential breaking chagnes.

For the second, why is this needed? The common-external.yaml creates the namespace. Why should the script create it? Is this in case the script is run before the yaml is created?

These two changes would be better in separate commits, with a complete description for the changes. The PR and commit descriptions don't currently provide any background.